services/object/src/api.js

```diff
@@ -10,4 +11,25 @@ let builder = new APIBuilder({
   context: ['cfg', 'db', 'backends'],
 });
 
+builder.declare({
+  method: 'post',
+  route: '/upload/:name/:projectId',
+  name: 'uploadObject',
+  input: undefined,
+  output: undefined,
+  stability: 'experimental',
+  category: 'Backend',
+  scopes: 'object:upload:<name>/<projectId>',
+  title: 'Upload backend data',
+  description: [
+    'Upload backend data.',
+  ].join('\n'),
+}, async function(req, res) {
+  let { name, projectId } = req.params;
+  const backend = this.backends.forUpload({ name, projectId });
+  let expires = fromNow('1 year');
+  await this.db.fns.upload_object(name, { backendId: backend.backendId, name, projectId }, expires);
```

Comment:
I suspect that we'll want to be able to query by backendId and by projectId in order to calculate costs.  Let's make those top-level column on the table, rather than JSON properties.  And, the name is already a column, so there's no need to include it in `data`.